pages/belt_docs/map-dict.mdx

```diff
@@ -1,272 +1,306 @@
 import { make as Layout } from "../../layouts/BeltDocsLayout.bs.js";
+import { make as ApiIntro } from "../../components/ApiIntro.bs.js";
 export default Layout;
 
 # MapDict
 
+<ApiIntro>
+
+This module seprate identity from data, it is a bit more verboe but slightly more efficient due to the fact that there is no need to pack identity and data back after each operation
+
+***Advanced usage only***
+
+</ApiIntro>
+
 ## t
 
-```re sig
```

Comment:
Why remove the `re sig`?

We plan to use it to apply special styling in the future.

Reply:
oh i looked at the list and none of them had it